data.tf

```diff
@@ -9,5 +9,11 @@ data "template_file" "asg_ecs_ec2_user_data" {
     ecs_cluster_name  = local.ecs_cluster_name
     env               = var.env
     ec2_service_group = var.ec2_service_group
+    auto_assign_eip   = var.ec2_asg_auto_assign_eip
```

Comment:
Can we use https://www.terraform.io/language/functions/tostring here and switch the whole var to bool?